packages/xo-web/src/xo-app/backup/overview-legacy/index.js

```diff
@@ -141,16 +141,26 @@ const JOB_COLUMNS = [
 
 // ===================================================================
 
-@addSubscriptions({
-  jobs: cb => subscribeJobs(jobs => cb(keyBy(jobs, 'id'))),
+@addSubscriptions(({ jobPredicate }) => ({
```

Comment:
I don't think we should bother showing legacy backups at all. Users should all be on Backup NG now and we don't need to support it more than the legacy table that already exists.
Maybe just show a warning in vm/backups if the VM is still attached to a legacy backup.